workflows/codefresh-csdp/versions/1.1.3/rbac.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: argo-hub.codefresh-csdp.1.1.3
```

Comment:
maybe @denis-codefresh is the correct person to address -- but why do we need to create a new service account each time with different version?